packages/sw-broadcast-cache-update/test/sw/behavior.js

```diff
@@ -0,0 +1,55 @@
+importScripts(
+  '/node_modules/mocha/mocha.js',
+  '/node_modules/chai/chai.js',
+  '/node_modules/sw-testing-helpers/build/browser/mocha-utils.js',
+  '/packages/sw-broadcast-cache-update/build/sw-broadcast-cache-update.min.js'
+);
+
+const expect = self.chai.expect;
+mocha.setup({
+  ui: 'bdd',
+  reporter: null,
+});
+
+describe('Test of the Behavior class', function() {
+  const channelName = 'test-channel';
+  const headersToCheck = ['one', 'two'];
+  const source = 'test-source';
+
+  it(`should throw when Behavior() is called without any parameters`, function() {
+    expect(() => new goog.broadcastCacheUpdate.Behavior()).to.throw();
```

Comment:
Could you define an error that is expected so that we know it's not some other error, for example browserCacheUpdate is undefined causing the error rather that the empty constructor causing the error..